Sources/SwiftDocC/Semantics/ExternalLinks/ExternalReferenceWalker.swift

```diff
@@ -174,21 +174,28 @@ struct ExternalReferenceWalker: SemanticVisitor {
     }
 
     mutating func visitComment(_ comment: Comment) { }
-    
+
+    mutating func visitMarkups(_ markups: [Markup]?) {
+        markups?.forEach { visitMarkup($0) }
+    }
+
     mutating func visitSymbol(_ symbol: Symbol) {
-        symbol.abstractSection.unwrap { visitMarkup($0.paragraph) }
-        symbol.discussion.unwrap { $0.content.forEach { visitMarkup($0) }}
-        symbol.topics.unwrap { $0.content.forEach { visitMarkup($0) }}
-        symbol.seeAlso.unwrap { $0.content.forEach { visitMarkup($0) }}
-        symbol.returnsSection.unwrap { $0.content.forEach { visitMarkup($0) }}
-        
-        symbol.parametersSection.unwrap {
-            $0.parameters.forEach {
-                $0.contents.forEach { visitMarkup($0) }
-            }
+        visitMarkups(symbol.abstractSection?.content)
```

Comment:
I wasn't sure either so I ran a benchmark comparison comparing this ("slow"):

```
visitSectionVariants(symbol.abstractSectionVariants)
```

...with this ("fast"):

```
for abstractSection in symbol.abstractSectionVariants.allValues.map(\.variant) {
    for markup in abstractSection.paragraph.children { visitMarkup(markup) }
}
```

There was no significant difference:


```
┌──────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Metric                                   │ Change          │ slow                 │ fast                 │
├──────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Duration for 'bundle-registration'       │ no change¹      │ 0.235 sec            │ 0.236 sec            │
│ Duration for 'convert-total-time'        │ no change²      │ 3.434 sec            │ 3.413 sec            │
│ Duration for 'documentation-processing'  │ no change³      │ 3.135 sec            │ 3.115 sec            │
│ Duration for 'finalize-navigation-index' │ no change⁴      │ 0.016 sec            │ 0.015 sec            │
│ Peak memory footprint                    │ no change⁵      │ 43.6 MB              │ 43.4 MB              │
│ Data subdirectory size                   │ no change⁶      │ 13.5 MB              │ 13.5 MB              │
│ Index subdirectory size                  │ no change⁷      │ 314 KB               │ 314 KB               │
│ Total DocC archive size                  │ no change⁸      │ 17.2 MB              │ 17.2 MB              │
│ Topic Anchor Checksum                    │ no change       │ 508d5249696fb765486f │ 508d5249696fb765486f │
│ Topic Graph Checksum                     │ no change       │ 2b0cd29ff867a49f008a │ 2b0cd29ff867a49f008a │
└──────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

So I decided to use the "slow" version since it's easier to read and understand.